### DISCUSSION TITLE
Is it possible to set default value for belongs_to field?

### DISCUSSION THREAD

**Discussion by xeron** [avo-hq/avo#2640] [Help] [answered]
Trying the following but doesn't seem like it's working:

```ruby
field :season, as: :belongs_to, default: -> { Season.first }
```

I also tried the following as an experiment but got a error:

```ruby
    field :season, as: :select, options: -> { Season.all }, default: -> { Season.first },
      format_using: -> { value.name }
```

```
undefined method `invert' for #<ActiveRecord::Relation [#<Season id ... >]>
```

Am I doing something wrong or `belongs_to` just doesn't support `default`? Latest avo version.

**Comment by adrianthedev** [comment, marked as answer]
Try `Season.first.id.to_s` That should do it.

**Comment by xeron** [comment]
> ```ruby
> field :season, as: :belongs_to, default: -> { Season.first }
> ```

Thanks! This worked. And it worked w/o `to_s` as well, probably because I'm using UUID IDs.

**Reply by adrianthedev** [reply]
Nice! I'm glad it worked!
